### PR TITLE
Updated README.md file to have a working example.

### PR DESCRIPTION
**Pull request by YesSeri**
I did this to make the example in README work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ npm install @lubert/node-uci-engine
 ## Usage
 
 ```javascript
-const { Engine, Position, SearchConfig } = require('@lubert/node-uci-engine');
+const { Engine } = require('@lubert/node-uci-engine');
 
 const engine = new Engine('/path/to/your/engine');
-const position = new Position("r1bqkb1r/5ppp/p2ppn2/1pn5/3NP3/1BN5/PPP2PPP/R1BQR1K1 w kq - 4 10");
-const config = new SearchConfig({ depth: 15 });
+const position =  { fen:"r1bqkb1r/5ppp/p2ppn2/1pn5/3NP3/1BN5/PPP2PPP/R1BQR1K1 w kq - 4 10") };
```
Comment (lubert) [inline README.md:20]: Can you remove the extra space before the bracket on this line and the next?
Also, can you add a space after `fen:`

**Review by lubert — CHANGES_REQUESTED** [review]
Thanks for fixing the documentation, couple of small nits

**Comment by YesSeri** [comment]
I have updated the my fork, and I think I have made a pull request. I removed a parenthesis that made the example not work, and fixed the things you mentioned. Have I made a new pull request correctly?

**Comment by lubert** [comment]
> I have updated the my fork, and I think I have made a pull request. I removed a parenthesis that made the example not work, and fixed the things you mentioned. Have I made a new pull request correctly?

Yep, looks good to me

**Review by lubert — APPROVED** [review]

**Pull request merged**